layers/+lang/typescript/packages.el

```diff
@@ -18,13 +18,16 @@
         tide
         typescript-mode
         web-mode
+        yasnippet
         ))
 
 (defun typescript/post-init-add-node-modules-path ()
   (add-hook 'typescript-mode-hook #'add-node-modules-path)
   (add-hook 'typescript-tsx-mode-hook #'add-node-modules-path))
 
 (defun typescript/post-init-company ()
+  (add-hook 'typescript-mode-hook #'spacemacs/typescript-company-setup)
+  (add-hook 'typescript-tsx-mode-hook #'spacemacs/typescript-company-setup)
   (spacemacs|add-company-backends
     :backends company-tide
     :modes typescript-mode typescript-tsx-mode))
```

Comment:
I just learned this from another layer, but I believe you can replace the hooks above with `:variables company-minimum-prefix-length 2` in the `spacemacs|add-company-backends` call. Look for the other places that `company-minimum-prefix` is set in spacemacs.

Reply:
@aaronjensen done